File: ExercicioCozinha/Cozinha.java
package SquadraNewThinkers_1T2021.ExercicioCozinha;

import java.util.ArrayList;

public class Cozinha{
  private String tipo;
  private int numeroPratos;
  private int numeroCozinheiros;
  private int tempoPreparo;
  private int horaAbertura;
  private int horaFechamento;
  private String pratoPrincipal;
  private ArrayList<Ingrediente> ingredientes;
  private ArrayList<Funcionario> funcionarios;

  public Cozinha(String tipo, int numPratos, int numCozinheiros,int tempoPreparo,
  int hAbertura, int hFechamento, String pratoPrincipal){
    this.tipo = tipo;
    this.numeroPratos = numPratos;
    this.numeroCozinheiros = numCozinheiros;
    this.tempoPreparo = tempoPreparo;
    this.horaAbertura = hAbertura;
    this.horaFechamento = hFechamento;
    this.pratoPrincipal = pratoPrincipal;
    ingredientes = new ArrayList<Ingrediente>();
    funcionarios = new ArrayList<Funcionario>();
  }
  
  public void adicionarIngrediente(Ingrediente i){
    ingredientes.add(i);
  }

  public void adicionarFuncionarios(Funcionario f){
    funcionarios.add(f);
  }
  
  public void imprimirDadosCozinha() {
    System.out.println("\nRestaurante Cozinha: " + tipo + "\nNúmero de Pratos: " + numeroPratos + 
    "\nNúmero de Cozinheiros: " + numeroCozinheiros + "\nTempo de Preparo: " +
    tempoPreparo + "min\nHora de Abertura: " + horaAbertura + "h\nHora de Fechamento: " +
    horaFechamento + "h\nPrato Principal: " + pratoPrincipal);
    imprimirIngredientes();
    imprimirFuncionarios();
    System.out.println("\n=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=");
  }

  public void imprimirIngredientes(){
    System.out.println("\n>>> Ingredientes <<<");
    for(Ingrediente i: ingredientes) {
      System.out.println(i);
    }
  }

  public void imprimirFuncionarios(){
    System.out.println("\n>>> Funcionarios <<<");
    for(Funcionario f: funcionarios){
      System.out.println(f);
    }
  }
}

File: PHP_atividade1/classes/Cozinha.php
<?php
require_once "Funcionario.php";
require_once "Ingrediente.php";

class Cozinha
{
    private string $tipo;
    private int $numeroPratos;
    private int $numeroCozinheiros;
    private int $tempoPreparo;
    private int $horaAbertura;
    private int $horaFechamento;
    private string $pratoPrincipal;
    private array $ingredientes = [];
    private array $funcionarios = [];

    public function __construct(string $tipo, int $numeroPratos, int $numeroCozinheiros, int $tempoPreparo, int $horaAbertura, int $horaFechamento, string $pratoPrincipal)
    {
        $this->tipo = $tipo;
        $this->numeroPratos = $numeroPratos;
        $this->numeroCozinheiros = $numeroCozinheiros;
        $this->tempoPreparo = $tempoPreparo;
        $this->horaAbertura = $horaAbertura;
        $this->horaFechamento = $horaFechamento;
        $this->pratoPrincipal = $pratoPrincipal;
    }

    public function getTipo(): string
    {
        return $this->tipo;
    }

    public function getNumeroPratos(): int
    {
        return $this->numeroPratos;
    }

    public function getNumeroCozinheiros(): int
    {
        return $this->numeroCozinheiros;
    }

    public function getTempoPreparo(): int
    {
        return $this->tempoPreparo;
    }

    public function getHoraAbertura(): int
    {
        return $this->horaAbertura;
    }

    public function getHoraFechamento(): int
    {
        return $this->horaFechamento;
    }

    public function getPratoPrincipal(): string
    {
        return $this->pratoPrincipal;
    }

    public function getIngredientes(): array
    {
        return $this->ingredientes;
    }

    public function setIngredientes(Ingrediente $ingrediente): void
    {
        array_push($this->ingredientes, $ingrediente);
    }

    public function getFuncionarios(): array
    {
        return $this->funcionarios;
    }

    public function setFuncionarios(Funcionario $funcionario): void
    {
        array_push($this->funcionarios, $funcionario);
    }
}